nectar/src/swap/comit.rs

```diff
@@ -8,3 +8,18 @@ mod herc20_hbit;
 pub use comit::{ethereum, *};
 pub use hbit_herc20::{hbit_herc20_alice, hbit_herc20_bob};
 pub use herc20_hbit::{herc20_hbit_alice, herc20_hbit_bob};
+
+use std::fmt::Debug;
```

Comment:
Nectar is in `comit-rs`. If that "could be upstreamed" why not upstreaming it directly? Feels very limbo-ish.

Reply:
Yeah, I am planning to do that at some point, just didn't want to add even more noise to these PRs :D